bootstrap/kubeadm/controllers/kubeadmconfig_controller_test.go

```diff
@@ -924,6 +924,149 @@ func TestBootstrapTokenTTLExtension(t *testing.T) {
 	}
 }
 
+func TestBootstrapTokenRotationMachinePool(t *testing.T) {
+	t.Skip("This now fails because it's using Update instead of patch, needs rework")
```

Comment:
TODO: need to fix this

test is passing in release-0.3 branch, same as https://github.com/kubernetes-sigs/cluster-api/pull/3940/files#diff-ad5b3c5b9e092b1dced2a9f66a655d659462a11bd6138e7c3f9853723f924f9aL777